.github/workflows/ci.yml

```diff
@@ -73,4 +73,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: --all-targets --all-features -- -D warnings -A clippy::type_complexity
+          args: --all-targets --all-features -- -D warnings -D clippy::type_complexity
```

Comment:
I don't think you have to specify the `-D` flag here, `type_complexity` is enabled by default.

Reply:
Agrred.